rholang/src/main/scala/coop/rchain/rholang/interpreter/Substitute.scala

```diff
@@ -116,32 +116,26 @@ object Substitute {
                 case Left(_e)    => par.prepend(_e)
                 case Right(_par) => _par ++ par
               }
+            case e @ EEvalBody(_) =>
```

Comment:
if you don't care about value inside you can just type `case e: EEvalBody`. I'm fine with both